test/functional/unified-spec-runner/operations.ts

```diff
@@ -219,6 +241,14 @@ operations.set('createCollection', async ({ entities, operation }) => {
   });
 });
 
+operations.set('createFindCursor', async ({ entities, operation }) => {
+  const collection = entities.getEntity('collection', operation.object);
+  const { filter, sort, batchSize, limit, let: vars } = operation.arguments;
+  const cursor = collection.find(filter, { sort, batchSize, limit, let: vars });
+  await cursor.hasNext();
```

Comment:
I assume this is a little hack to make it grab the first batch? Let's leave a comment

Reply:
Yes it needs to execute the command but not step the cursor forward. I'll add a comment.

Reply:
Added comment.